lib/cmds/start.ts

```diff
@@ -220,6 +220,12 @@ function start(options: Options) {
 
           args.push('-jar');
           args.push(path.resolve(outputDir, binaries[Standalone.id].filename()));
+          args.push('-role');
```

Comment:
Should these args be behind the detach flag? If this is the default behavior, it will have an exception in the logs since there is no role=hub.